kube/pod.go

```diff
@@ -370,6 +370,15 @@ func getEnvVarsFromConfigs(configs model.ConfigurationVariableSlice, settings Ex
 			continue
 		}
 
+		if config.Name == "HELM_IS_INSTALL" {
+			value := "true"
```

Comment:
That's the string `true`, not the boolean value `true`, right? That's… actually probably the right thing to do in an environment variable, I guess.

Reply:
Yes, env vars are always strings.

Reply:
Now the real question is: should this be `true` or `false` in a kube config? I guess it doesn't really matter, as it should not be used by the bosh-tasks. So maybe it would be better not to set it at all in that case. Not sure any of this matters; I hope the pure kube config files go away at some point...

Reply:
I'm not too worried about the kube versions; they're already busted  (`external_ips` is a string instead of a list), and nobody complained. (I'm fixing it in #354 anyway).

Whatever value you put here is fine; even `unknown` or `pants`.